source/assets/js/playground.ts

```diff
@@ -193,15 +194,53 @@ function setupPlayground(): void {
     const copyURLButton = document.getElementById('playground-copy-url');
     const copiedAlert = document.getElementById('playground-copied-alert');
 
-    let timer: undefined | number;
+    type Timer = undefined | number;
+
+    let alertTimer: Timer;
+    const buttonTimers: {input: Timer; output: Timer; url: Timer} = {
+      input: undefined,
+      output: undefined,
+      url: undefined,
+    };
+
+    function showCopiedAlert(msg: string): void {
+      if (!copiedAlert) return;
+      copiedAlert.innerText = msg;
+      copiedAlert.classList.add('show');
+      if (alertTimer) clearTimeout(alertTimer);
+      alertTimer = window.setTimeout(() => {
+        copiedAlert.classList.remove('show');
+      }, 3000);
```

Comment:
This this number is used in multiple places now, it's probably a good idea to make it a top-level constant.

Reply:
Resolved in https://github.com/sass/sass-site/pull/1177/commits/661dfef9188f7dccc54ad4b78209526cefadb94d